amethyst_renderer/src/sprite.rs

```diff
@@ -183,14 +248,70 @@ impl<'a> WithSpriteRender for EntityBuilder<'a> {
     }
 }
 
+/// Sprite sheets used by sprite render animations
```

Comment:
Do you mind adding a bit more doc? Thanks!

Reply:
haha, shall!

* [ ] more doc = good